app/models/group.rb

```diff
@@ -47,6 +47,35 @@ def extend_trial
     self.save
   end
 
+  def ensure_group_user_exist()
+    uid = %(group@group-#{id}.co)
+    group_user = User.find_by uid: uid
+    if !group_user 
+      group_user = User.create!({
+          name: %(Group "#{name}"),
+          uid: uid,
+          email: uid,
```

Comment:
email ends up being something like "group@group-315.co" 
I feel like this is a little bit strange bc this looks like a real email address but isn't..

Reply:
Fair point. The reason is the User model is quite restrictive with the shape of Emails. It actually has to look like a real email address. We could of course loosen that restriction or move the validation to only be relevant for emails entered by users.

I get that the idea of using existing users and memberships for the group account is kind of cheating and it feels a bit like forcing different concepts into a single representation.

What would be your preferred solution here?

Reply:
hmmm how about we do group-315@cobudget.co ?